lms/templates/inline_analytics.html

```diff
@@ -42,12 +45,14 @@
       <div id="${part_id}_analytics" class="${element_id}_analytics_div inline-analytics-item" data-correct-response="${correct_response}" data-question-type="${question_type}">
         <section aria-hidden="true" >
           <div class="grid-wrapper checkbox">
-            <p>Answer distribution of the <strong class="num-students"></strong> students who answered this question:</p>
-            <p>Last updated: <strong class="last-update"></strong></p>
+            <p>
+              ${_('{title}').format(title=u'Answer distribution of the <strong class="num-students"></strong> students who answered this question.')}
```

Comment:
`${_('Answer distribution of the {num_students} students who answered this question.').format(num_students='<strong class="num-students"></strong>')}`